clifford/test/test_trig_functions.py

```diff
@@ -0,0 +1,86 @@
+
+from .. import Cl
```

Comment:
Relative imports in tests are usually a bad idea - tests shouldn't assume they are part of the package.

```suggestion
from clifford import Cl
```